packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -1405,7 +1405,7 @@ export class BasicCrawler<
             },
             pushData: this.pushData.bind(this),
             useState: this.useState.bind(this),
-            sendRequest: createSendRequest(this.httpClient, request!, session),
+            sendRequest: createSendRequest(this.httpClient, request!, session) as CrawlingContext['sendRequest'],
```

Comment:
I have no problem with adding a cast, but I can't help myself asking... what is the issue?